app/views/business_mailer/subscribed.text.erb

```diff
@@ -0,0 +1,15 @@
+<%= @business.first_name %>,
+
+It’s Joe again, the founder of RailsDevs. Your subscription is live and you can message all <%= @developer_count %>+ Rails developers on the platform.
+
+Get started by adding a filter or two when <%= link_to "browsing developer profiles", developers_url %>. Start with role level and work preference then try adding a specific term or technology.
+
+When you see a good fit click the Hire button. Your message goes right to their email inbox.
+
+Hit reply or <%= link_to "schedule a call", "https://savvycal.com/joemasilotti/RailsDevs" %> if you’d like help narrowing down your search!
```

Comment:
These won't render in a plain text email. Please move the link to the end of the line like in `BusinessMailer#welcome`.